src/newcharacter.cpp

```diff
@@ -1887,8 +1901,11 @@ tab_direction set_scenario(WINDOW *w, player *u, points_left &points)
                   sorted_scens[cur_id]->gender_appropriate_name(u->male).c_str(),
                   pointsForScen);
 
-        fold_and_print(w_description, 0, 0, TERMX - 2, c_green,
-                       sorted_scens[cur_id]->description(u->male).c_str());
+        std::string scenUnavailable = "This scenario is not available in this world due to city size settings. ";
+        std::string scenDesc = ( sorted_scens[cur_id]->has_flag( "CITY_START" ) && !scenario_sorter.cities_enabled ) ? 
+            (scenUnavailable + sorted_scens[cur_id]->description( u->male ).c_str()) :
+            sorted_scens[cur_id]->description( u->male ).c_str();
```

Comment:
Why the call to `c_str` here? It should work fine without it. Same in the line above.